src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1363,21 +1363,28 @@ because xbuild doesn't support framework reference assemblies.
 	</ComputeHash>
 </Target>
 
-<Target Name="_CompileAndroidLibraryResources" DependsOnTargets="_CollectLibraryResourceDirectories"
+<Target Name="_ConvertLibraryResourcesCases"
 		Condition="'$(_AndroidUseAapt2)' == 'True'"
```

Comment:
Shouldn't this Target depend on `_CollectLibraryResourceDirectories`?

Reply:
This is setup on the other target:
```xml
<Target Name="_CompileAndroidLibraryResources" DependsOnTargets="_CollectLibraryResourceDirectories;_ConvertLibraryResourcesCases"
```
I'm not sure if one way is more correct? Do we have a preference?

Reply:
I think they're equally "correct", execution-wise, but it's also a question of *maintenance* and *testing*.

For example, as-is *this* will fail:

```bash
$ msbuild App.csproj /t:_CompileAndroidLibraryResources
```

It'll fail because `@(_LibraryResourceHashDirectories.FileFound)` only exists once `_CollectLibraryResourceDirectories` has been executed.

Now, is this *important*? ¯\_(ツ)_/¯ 

I just think it might be useful for one-off testing/development.